roles/docker_image_build/vars/main.yml

```
@@ -14,6 +14,8 @@ run_commands: "{{ role_run_commands + common_run_commands + host_run_commands }}
 
 build_variants: "{{ role_build_variants + common_build_variants + host_build_variants }}"
 
+docker_labels: "{{ role_docker_labels | combine(common_docker_labels) | combine(host_docker_labels) }}"
```

My vague recollection is that `combine` is for dictionaries.

```suggestion
docker_labels: "{{ role_docker_labels + common_docker_labels + host_docker_labels }}"
```